discovery/gossiper.go

```diff
@@ -335,7 +359,7 @@ func New(cfg Config, selfKey *btcec.PublicKey) *AuthenticatedGossiper {
 		prematureChannelUpdates: make(map[uint64][]*networkMsg),
 		channelMtx:              multimutex.NewMutex(),
 		recentRejects:           make(map[uint64]struct{}),
-		heightForLastChanUpdate: make(map[uint64][2]uint32),
+		chanUpdateRateLimiter:   make(map[uint64][2]*rate.Limiter),
```

Comment:
Commenting here as can't below: so we're confident the keep-alive code isn't at fault, right?

Reply:
Yeah, so far the reports have been regarding updates that actually change something (e.g., disabled bit). Keep alive updates can't be done by the user at will within lnd, they're only sent every 24H.